kademlia/network.py

```diff
@@ -20,6 +20,8 @@ class Server(object):
     to start listening as an active node on the network.
     """
 
+    protocol_class = KademliaProtocol
```

Comment:
It's a small thing - but just for consistency with Twisted (and given how much is copied over from Twisted into asyncio, I think it's worth noting) - I think this should just be `protocol` instead of `protocol_class`.  What do you think?